src/lib.rs

```diff
@@ -2533,6 +2533,21 @@ impl Build {
                 }
                 None => default_ar,
             }
+        } else if target.starts_with("wasm32") {
+            // Formally speaking one should be able to use this approach,
+            // parsing -print-search-dirs output, to cover all clang targets,
+            // including Android SDKs and other cross-compilation scenarios...
+            // And even extend it to gcc targets by seaching for "ar" instead
+            // of "llvm-ar"...
+            let compiler = self.get_base_compiler()?;
+            if compiler.family == ToolFamily::Clang {
```

Comment:
Coudl this condition be moved up into the `if target.starts_with("wasm32")` condition?